python/mxnet/libinfo.py

```diff
@@ -29,6 +29,9 @@ def find_lib_path():
         dll_path.extend([p.strip() for p in os.environ['LD_LIBRARY_PATH'].split(":")])
     if os.name == 'nt':
         dll_path = [os.path.join(p, 'libmxnet.dll') for p in dll_path]
+    elif platform.system() == 'Darwin':
+        dll_path = [os.path.join(p, 'libmxnet.dylib') for p in dll_path]+ \
```

Comment:
When building wheel on OSX, I didn't have to include dylib extension as the build process still produced .so.
Has there been any changes that made this necessary? Even if it's just related to amalgamation, it would still be helpful to include relevant information to help me understand. Thanks.